jaeger-core/src/main/java/com/uber/jaeger/samplers/Sampler.java

```diff
@@ -29,4 +29,8 @@
    * Release any resources used by the sampler.
    */
   void close();
+
+  interface Builder {
```

Comment:
why is this needed?

Reply:
The reason is mentioned in PR description. Some samplers are easy to create, however some (like `RemoteControlledSampler`) has several configuration options therefore there is a generic builder.

Reply:
when someone creates a concrete sampler/reporter, they are dealing with a concrete class. If it happened to have a builder, it would be specific to that class. So I am not seeing the value of a common builder interface, unless the intention is to pass builders around.